models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -15,19 +15,22 @@
 ##' @author Michael Dietze
 write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs = NULL, IC = NULL,
                                 restart = NULL, spinup = NULL) {
+  
   ### WRITE sipnet.in
   template.in <- system.file("sipnet.in", package = "PEcAn.SIPNET")
   config.text <- readLines(con = template.in, n = -1)
   writeLines(config.text, con = file.path(settings$rundir, run.id, "sipnet.in"))
 
   ### WRITE *.clim
-  template.clim <- settings$run$input$met$path  ## read from settings #typo in inputs?
-  if (!is.null(inputs)) {
-    ## override if specified in inputs
-    if ("met" %in% names(inputs)) {
-      template.clim <- inputs$met$path
-    }
+  template.clim <- settings$run$inputs$met$path$path1
```

Comment:
we should not be getting input paths in this format

Reply:
The decision to add list depth and use a list instead of a vector was discussed here: https://pecanproject.slack.com/archives/C8RD9BRLK/p1531508672000023?thread_ts=1531504235.000165&cid=C8RD9BRLK .  All input paths were wrapped in an additional list, even if there was only one, for consistency.  This was part of pull request #2004 .  

The path settings remain(ed) unchanged throughout the workflow, and write.configs.SIPNET was changed to reflect this.

Is it the responsibility of `met.process` to return file paths in settings differently, or is it the responsibility of `run.write.configs` to format the arguments differently? (Relevant thread here: https://pecanproject.slack.com/archives/C8RD9BRLK/p1533301225000306?thread_ts=1533246209.000194&cid=C8RD9BRLK).

Reply:
I understand that met.process and the other do.conversions functions will be generating ensemble Inputs that have multiple paths and multiple ID's, but it is the responsibility of `run.write.configs` to pass every `write.configs.[model]` a single specific path for every Input, and that input is expected to be at `settings$run$inputs$<input>$path`. This change doesn't just break SIPNET, it breaks every single model and add unnecessary complexity/confusion to write.configs.[model]. No model needs to know whether it's being run individually or in an ensemble or which ensemble member it is.

Reply:
This code happened to be causing a merge conflict anyway, so I resolved the conflict in favor of the develop version.  So long as the changes to `run.write.configs` in #2025 send a single input path at `settings$run$inputs$<input>$path`, this should be good to go.